lib/translations/app_localizations.dart

```diff
@@ -0,0 +1,291 @@
+import 'package:flutter/material.dart';
+
+class AppLocalizations {
+  final Locale locale;
+
+  AppLocalizations(this.locale);
+
+  static AppLocalizations of(BuildContext context) {
+    return Localizations.of<AppLocalizations>(context, AppLocalizations);
+  }
+
+  static Map<String, Map<String, String>> _localizedValues = {
```

Comment:
can we have this in a file so:
when the app starts file is loaded so in memory we have only one language loaded and if we wish to add another language we just add a new file with translations?
`default.language.txt`
```
loginScreen.connect:->CONNECT
loginScreen.add.new.connection:->ADD NEW CONNECTION
....
```

Reply:
I could write jsons for different languages but changing this structure that Bartek created will cost at least few hours of work and even i I did so in main:
` supportedLocales: [
          const Locale('en', ''),
        ],`
and then in AppLocalizationsDelegate
`@override
  Future<AppLocalizations> load(Locale locale) {
    return SynchronousFuture<AppLocalizations>(AppLocalizations(locale));
  }`
all the supported languages must be loaded at once so there would not be a better performance.

Reply:
Ok just make sure that all user-facing LABELS are in one place so they will be easy to refactor in the future.

Reply:
Let's make this simpler - as we only have one language for now.
Please make this a flat map `<String, String>` for now.
Rename `_localizedValues` to `_default`

Just key-value pairs - no `en`

Reply:
![image](https://user-images.githubusercontent.com/36708034/105822237-238f0d00-5fbc-11eb-8d52-d45584968aa9.png)
but we have followed the docs doing that
https://flutter.dev/docs/development/accessibility-and-localization/internationalization
'en' key is needed and is also used in for instance picking supported languages:
`@override
  bool isSupported(Locale locale) {
    return ['en'].contains(locale.languageCode);
  }`
and is declared in main.dart so I think this is the structure we have to and should follow.

Reply:
And the given example makes sense - look at your version.
in your version: get property first then get language
in the example: get language first ane then get prop

it should be reimplemented to this:
```
{ 'en': {
  'loginScreen.connect': 'CONNECT',
  'loginScreen.add.new.connection': 'New Connection',
   ...........
  }
}
```

Reply:
💯 I will fix this for you